### ISSUE TITLE
Inconsistency between INPUT and output for atom magnetism in ferromagnetic calculation.

### ISSUE THREAD

**Issue by hongriTianqi** [deepmodeling/abacus-develop#520]
## Describe the Bug
The magnetism for Fe atoms are set to 5.0 in STRU file, but are show as 0.0 in output file when initialing calculations. 

## To Reproduce
1. cat 104_PW_FM_magnetic/STRU
2. cat 104_PW_FM_magnetic/OUT.autotest/running_scf.log

[104_PW_FM_magnetic.tar.gz](https://github.com/deepmodeling/abacus-develop/files/7640378/104_PW_FM_magnetic.tar.gz)

**Comment by dyzheng** [comment]
Yes, this is a remaining problem. 
In ABACUS, initial magnetism can start from two method:

1. magnetism for every element.
2. magnetism for every atom.

These two method related to different code, and method 2 has a higher priority than method 1. 
Unfortunately, we found there is a bug in relax calculation with method 2, so we set default value of "mag" as zero in order to use method 1 as default method.
